packages/fxa-auth-server/test/remote/oauth_tests.js

```diff
@@ -312,4 +312,52 @@ describe('/oauth/ routes', function() {
     const clientRotatedJWT = decodeJWT(clientRotatedRes.access_token);
     assert.notEqual(tokenJWT.claims.sub, clientRotatedJWT.claims.sub);
   });
+
+  it('successfully revokes access tokens, and refresh tokens', async () => {
+    let res = await client.createAuthorizationCode({
+      client_id: PUBLIC_CLIENT_ID,
+      state: 'abc',
+      code_challenge: MOCK_CODE_CHALLENGE,
+      code_challenge_method: 'S256',
+      scope: 'profile openid',
+      access_type: 'offline',
+    });
+    assert.ok(res.code);
+
+    res = await client.grantOAuthTokens({
+      client_id: PUBLIC_CLIENT_ID,
+      code: res.code,
+      code_verifier: MOCK_CODE_VERIFIER,
+    });
+    assert.ok(res.access_token);
+    assert.ok(res.refresh_token);
+
+    await client.revokeOAuthToken({
+      client_id: PUBLIC_CLIENT_ID,
+      token: res.access_token,
+    });
```

Comment:
Is it possible to add a test for a JWT access token? Needed?

Reply:
Added, and in fact this uncovered an unhandled error case - if you pass in an invalid JWT access token, we would try to destroy it as a refreshToken and would fail internal route validation.